src/settings.py

```diff
@@ -1,7 +1,6 @@
 import pygame
 import pygame.freetype
-import pytmx
-from src.import_checks import *
+import pytmxfrom src.import_checks import *
```

Comment:
Whoopsie! I'm sure you know what the problem is here. Typos, amirite?

Reply:
woops